### PR TITLE
feat(config): add Spectrum Brands 893 deadbolt

### PR DESCRIPTION
**Pull request by Botched1**
Add Spectrum Brands 893 device configuration. This is exactly the same as the Spectrum Brands 892 already in the database, but this version is zwave 700.

Took the 892 file and changed productId, zwaveAllianceId, and link to the manual.

### CODE REVIEW

**Comment by Botched1** [comment]
Linked to issue: #3702

**Comment by AlCalzone** [comment]
Since you merely copied the other file, I'm okay with this. The original file (and by extension this one) doesn't match our style guide however. Would you be willing to help clean it up? I can give you some pointers then.

**Comment by Botched1** [comment]
> Since you merely copied the other file, I'm okay with this. The original file (and by extension this one) doesn't match our style guide however. Would you be willing to help clean it up? I can give you some pointers then.

Sure. I don't mind cleaning the new one, or both, up if you have some things you have in mind.

File: packages/config/config/devices/0x0090/893.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Status LED Control",
+			"description": "The configuration parameter 1 is a read/write parameter with a one-byte field, used to control the status led. if set to on, the status LED will show the status of the latch every 6 seconds. if set to off, the status LED will not be shown",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]: This description seems unnecessary. If anything you might mention the 6 second interval.

File: packages/config/config/devices/0x0090/893.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Turns status LED off",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]: Disable

File: packages/config/config/devices/0x0090/893.json [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+					"value": 0
+				},
+				{
+					"label": "Turns status LED on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]: Enable

File: packages/config/config/devices/0x0090/893.json [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Sound Control",
+			"description": "The configuration parameter 2 is a read/write parameter with a one-byte field, used to control the lock’s buzzer. if set to on, the buzzer is enabled and will sound during normal events. if set to off, the buzzer is disabled, and the buzzer sound will not occur",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]:
```suggestion
			"label": "Buzzer",
```

File: packages/config/config/devices/0x0090/893.json [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+			"options": [
+				{
+					"label": "Turns the sound (buzzer) off",
+					"value": 0
+				},
+				{
+					"label": "Turns the sound (buzzer) on",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0090/893.json]: ...and so on. See https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide for all rules

File: packages/config/config/devices/0x0090/893.json [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disables secure screen (touch products only)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]: "touch products only" would be a good candidate for the description

File: packages/config/config/devices/0x0090/893.json (hunk 4 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0090/893.json]: and so on...

File: packages/config/config/devices/0x0090/893.json [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+				{
+					"label": "Left handed lock (read only)",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "40",
+			"label": "Factory Default Control",
+			"description": "The configuration parameter 40 is a write only parameter with a one-byte field, used to request the lock to perform a factory reset back to its default state. this control will cause the lock to be removed from the network, placing the lock back to its default state. when a read is issued for this parameter, a 0 will be returned",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]:
```suggestion
			"label": "Reset to Factory Default Setting",
```

File: packages/config/config/devices/0x0090/893.json [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+					"value": 0
+				},
+				{
+					"label": "Perform a full factory reset (set)",
+					"value": 1
+				},
+				{
+					"label": "Perform a modified factory reset (set) - for production only",
+					"value": 2
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]:
```suggestion
				{
					"label": "Normal Operation",
					"value": 0
				},
				{
					"label": "Reset to Factory Default",
					"value": 1
				},
				{
					"label": "Perform a modified Factory Reset",
					"value": 2
				}
```
(whatever the modified factory reset is)

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Here are several things I'd change - please note that the label and description changes are necessary for almost all params, even if not explicitly highlighted.

**Review by Botched1 — COMMENTED** [review]
This should be closer.

**Comment by Botched1** [comment]
@AlCalzone 
I re-did the file. I think it is a lot closer to what the style guide requests.

File: packages/config/config/devices/0x0090/893.json [hunk 9]
```diff
@@ -17,8 +17,8 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Status LED Control",
-			"description": "The configuration parameter 1 is a read/write parameter with a one-byte field, used to control the status led. if set to on, the status LED will show the status of the latch every 6 seconds. if set to off, the status LED will not be shown",
+			"label": "Status LED",
+			"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/893.json]: No empty description please. Remove entirely.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by Botched1 — COMMENTED** [review]
Getting closer! Removed the blank descriptions.

**Review by AlCalzone — APPROVED** [review]
Looks good, I just mentioned the label for the option to initiate direction detection in the description, since it might not be obvious.

**Comment by AlCalzone** [comment]
Could you do the same for the 892?

**Comment by Botched1** [comment]
> Could you do the same for the 892?

You bet. I should be able to submit that today.

**Pull request merged**